tf_agents/agents/dqn/examples/v2/train_eval.py

```diff
@@ -160,7 +165,20 @@ def train_eval(
           output_fc_layer_params,
           num_actions)
     else:
-      q_net = create_feedforward_network(fc_layer_params, num_actions)
+      observation_spec = tf_env.time_step_spec().observation
+      action_spec = tf_env.action_spec()
+      if network_type == 'q_network':
```

Comment:
what's the rationale behind adding this back? We've moved from using QNetwork to using sequential.Sequential (with create_feedforward_network)

Reply:
This acts as a side by side option comparison as the duel q network.  Could you please share with me why moving from using QNetwork to sequential?

Reply:
I was referring to this method here: https://github.com/tensorflow/agents/blob/54aaac0033985f17ffbf829af4e1487c8f43fd68/tf_agents/agents/dqn/examples/v2/train_eval.py#L341

Reply:
please see the commit : ab7bee2